pages/common/crontab.md

```diff
@@ -11,6 +11,10 @@
 
 `crontab -l`
 
+- View a list of existing cron jobs for every user:
+
+`for user in $(cut -f1 -d: /etc/passwd); do echo $user; crontab -u $user -l; done`
```

Comment:
We refrain from adding snippets or cheat-sheet like items like these. Our aim is to make the user understand how the command works, and then let him/her build more complex commands out of it.

Perhaps you can just mention `crontab -u {{user}}` and explain its functionality ?

Reply:
Thanks for your reply! Would make sense.